qemu/tests/pci_bridge_hotplug_unplug.py

```diff
@@ -0,0 +1,200 @@
+import logging
+
+from avocado.core import exceptions
+
+from virttest import error_context
+from virttest.qemu_devices import qdevices
+from virttest import storage
+from virttest import data_dir
+from virttest import utils_test
+
+
+def hotplug_virtio_blk_disk(vm, image_filename, parent_bus, image_extra_params):
+    drive = qdevices.QRHDrive("blk")
+    drive.set_param("file", image_filename)
+    drive.set_param("format", "qcow2")
+    drive_id = drive.get_param("id")
+    drive.hotplug(vm.monitor)
+
+    extra_params = image_extra_params.split("=")
+    vio_blk_params = {"id": "virtio_blk",
+                      "drive": drive_id,
+                      extra_params[0]: extra_params[1],
+                      "bus": parent_bus,
+                      "addr": 0x01}
+    device = qdevices.QDevice("virtio-blk-pci", params=vio_blk_params)
```

Comment:
I wonder why it's better to use qemu_devices.qcontainer instead of qemu_devices.qcontainer, Thanks! @luckyh